spec/fixtures/files/datahashes/anthro/claim1.json

```diff
@@ -3,11 +3,11 @@
     "nagpraClaimName": "Rucio",
     "nagpraClaimAltName": "one;two;",
     "nagpraClaimAltNameNote": "free;;available",
-    "claimFiledByPerson": "Tegla",
-    "claimFiledByOrganization": "chores",
-    "claimFiledOnBehalfOfPerson": "Tom",
-    "claimFiledOnBehalfOfOrganization": "Hola",
-    "claimantNote": "^^Moro^^Kioneki",
+    "claimFiledByPerson": "Tegla;Ann Analyst;Buck, Jane;;;;",
+    "claimFiledByOrganization": ";;;;chores;Organization 1;Organization 2",
+    "claimFiledOnBehalfOfPerson": "Tom;;Gabriel Solares;;;Ann Analyst;",
+    "claimFiledOnBehalfOfOrganization": ";Organization 1;;Hola;;;Organization 1",
+    "claimantNote": "Moro;n1;n2;;Kioneki;n3;n4",
```

Comment:
These are the problem fields and I've updated them with some more values and handled the delimiters in a way I think we might need to in order for this to work going forward. (Making this work is going to fail a bunch of the other tests, but so it goes)